src/sidebar/views/menu-item.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { DashboardMenuItemProps } from 'sidebar/sidebar.types';
+import { useTranslation } from 'react-i18next';
+function MenuItem({ props }: DashboardMenuItemProps) {
+  const { t } = useTranslation();
+  const ct = (text: string) => t(`common.${text}`);
+  const isExpanded = props.isExpanded(props.key);
+  return (
```

Comment:
Adding a new line before every `return` statement makes code cleaner and will be easy to find out whats being returned at what place.